test_0 (__main__.lab5a)
[Lab 5] - [Investigation 1] - [Part 1] - Files - Test for file creation: ./lab5a.py ... ok
test_a (__main__.lab5a)
[Lab 5] - [Investigation 1] - [Part 1] - Files - Test for errors running: ./lab5a.py ... ok
test_a1 (__main__.lab5a)
[Lab 5] - [Investigation 1] - [Part 1] - Files - Test for correct shebang line: ./lab5a.py ... ok
test_a_function_read_file_string (__main__.lab5a)
[Lab 5] - [Investigation 1] - [Part 1] - Files - function read_file_string fails without 1 argument ... ok
test_b_function_read_file_list (__main__.lab5a)
[Lab 5] - [Investigation 1] - [Part 1] - Files - function read_file_list fails without 1 argument ... ok
test_c_file_data_txt_exists (__main__.lab5a)
[Lab 5] - [Investigation 1] - [Part 1] - Files - Test for file creation: ./data.txt ... ok
test_d_file_data_txt_has_correct_text (__main__.lab5a)
[Lab 5] - [Investigation 1] - [Part 1] - Files - Test for correct contents: ./data.txt ... ok
test_e_function_read_file_string_correct_output (__main__.lab5a)
[Lab 5] - [Investigation 1] - [Part 1] - Files - Test for output: ./lab5a.py ... ok
test_f_function_read_file_list_correct_output (__main__.lab5a)
[Lab 5] - [Investigation 1] - [Part 1] - Files - Test for output: ./lab5a.py ... ok
test_g_function_read_file_list_correct_output (__main__.lab5a)
[Lab 5] - [Investigation 1] - [Part 1] - Files - Test for output: ./lab5a.py ... ok
test_h_function_read_file_list_correct_output (__main__.lab5a)
[Lab 5] - [Investigation 1] - [Part 1] - Files - Test for output: ./lab5a.py ... ok
test_0 (__main__.lab5b)
[Lab 5] - [Investigation 1] - [Part 2] - Files - Test for file creation: ./lab5b.py ... ok
test_a (__main__.lab5b)
[Lab 5] - [Investigation 1] - [Part 2] - Files - Test for errors running: ./lab5b.py ... ok
test_a1 (__main__.lab5b)
[Lab 5] - [Investigation 1] - [Part 2] - Files - Test for correct shebang line: ./lab5b.py ... ok
test_a_function_append_file_string (__main__.lab5b)
[Lab 5] - [Investigation 1] - [Part 2] - Files - function append_file_string fails without 2 argument ... ok
test_a_function_read_file_string (__main__.lab5b)
[Lab 5] - [Investigation 1] - [Part 2] - Files - function read_file_string fails without 1 argument ... ok
test_b_function_read_file_list (__main__.lab5b)
[Lab 5] - [Investigation 1] - [Part 2] - Files - function read_file_list fails without 1 argument ... ok
test_c_function_write_file_list (__main__.lab5b)
[Lab 5] - [Investigation 1] - [Part 2] - Files - function write_file_list fails without 2 argument ... ok
test_d_function_copy_file_add_line_numbers (__main__.lab5b)
[Lab 5] - [Investigation 1] - [Part 2] - Files - function copy_file_add_line_numbers fails without 2 argument ... ok
test_e_file_data_txt_exists (__main__.lab5b)
[Lab 5] - [Investigation 1] - [Part 2] - Files - Test for file creation: ./data.txt ... ok
test_f_file_data_txt_has_correct_text (__main__.lab5b)
[Lab 5] - [Investigation 1] - [Part 2] - Files - Test for correct contents: ./data.txt ... ok
test_g_function_read_file_string_correct_output (__main__.lab5b)
[Lab 5] - [Investigation 1] - [Part 2] - Files - Test for output: ./lab5b.py ... ok
test_h_function_read_file_list_correct_output (__main__.lab5b)
[Lab 5] - [Investigation 1] - [Part 2] - Files - Test for output: ./lab5b.py ... ok
test_i_function_append_file_string_correct_output_1 (__main__.lab5b)
[Lab 5] - [Investigation 1] - [Part 2] - Files - Test for output: ./lab5b.py ... /home/nyeh2/ops445/lab5/./CheckLab5.py:229: ResourceWarning: unclosed file <_io.TextIOWrapper name='seneca1.txt.checkscript' mode='a' encoding='UTF-8'>
  lab5bStudent.append_file_string(filename, string1)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
/home/nyeh2/ops445/lab5/./CheckLab5.py:230: ResourceWarning: unclosed file <_io.TextIOWrapper name='seneca1.txt.checkscript' mode='a' encoding='UTF-8'>
  lab5bStudent.append_file_string(filename, string1)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
ok
test_i_function_append_file_string_correct_output_2 (__main__.lab5b)
[Lab 5] - [Investigation 1] - [Part 2] - Files - Test for output: ./lab5b.py ... /home/nyeh2/ops445/lab5/./CheckLab5.py:245: ResourceWarning: unclosed file <_io.TextIOWrapper name='seneca1.txt.checkscript' mode='a' encoding='UTF-8'>
  lab5bStudent.append_file_string(filename, string1)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
/home/nyeh2/ops445/lab5/./CheckLab5.py:246: ResourceWarning: unclosed file <_io.TextIOWrapper name='seneca1.txt.checkscript' mode='a' encoding='UTF-8'>
  lab5bStudent.append_file_string(filename, string1)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
ok
test_j_function_write_file_list_correct_output_1 (__main__.lab5b)
[Lab 5] - [Investigation 1] - [Part 2] - Files - Test for output: ./lab5b.py ... /home/nyeh2/ops445/lab5/./CheckLab5.py:262: ResourceWarning: unclosed file <_io.TextIOWrapper name='seneca2.txt.checkscript' mode='w' encoding='UTF-8'>
  lab5bStudent.write_file_list(filename, list1)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
ok
test_j_function_write_file_list_correct_output_2 (__main__.lab5b)
[Lab 5] - [Investigation 1] - [Part 2] - Files - Test for output: ./lab5b.py ... /home/nyeh2/ops445/lab5/./CheckLab5.py:277: ResourceWarning: unclosed file <_io.TextIOWrapper name='seneca2.txt.checkscript' mode='w' encoding='UTF-8'>
  lab5bStudent.write_file_list(filename, list1)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
/home/nyeh2/ops445/lab5/./CheckLab5.py:278: ResourceWarning: unclosed file <_io.TextIOWrapper name='seneca2.txt.checkscript' mode='w' encoding='UTF-8'>
  lab5bStudent.write_file_list(filename, list1)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
ok
test_k_function_copy_file_add_line_numbers_correct_output_1 (__main__.lab5b)
[Lab 5] - [Investigation 1] - [Part 2] - Files - Test for output: ./lab5b.py ... /home/nyeh2/ops445/lab5/./CheckLab5.py:293: ResourceWarning: unclosed file <_io.TextIOWrapper name='seneca2.txt.checkscript' mode='r' encoding='UTF-8'>
  lab5bStudent.copy_file_add_line_numbers(filename, tempfile)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
/home/nyeh2/ops445/lab5/./CheckLab5.py:293: ResourceWarning: unclosed file <_io.TextIOWrapper name='seneca3.txt.checkscript' mode='w' encoding='UTF-8'>
  lab5bStudent.copy_file_add_line_numbers(filename, tempfile)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
ok
test_k_function_copy_file_add_line_numbers_correct_output_2 (__main__.lab5b)
[Lab 5] - [Investigation 1] - [Part 2] - Files - Test for output: ./lab5b.py ... /home/nyeh2/ops445/lab5/./CheckLab5.py:308: ResourceWarning: unclosed file <_io.TextIOWrapper name='seneca2.txt.checkscript' mode='r' encoding='UTF-8'>
  lab5bStudent.copy_file_add_line_numbers(filename, tempfile)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
/home/nyeh2/ops445/lab5/./CheckLab5.py:308: ResourceWarning: unclosed file <_io.TextIOWrapper name='seneca3.txt.checkscript' mode='w' encoding='UTF-8'>
  lab5bStudent.copy_file_add_line_numbers(filename, tempfile)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
/home/nyeh2/ops445/lab5/./CheckLab5.py:309: ResourceWarning: unclosed file <_io.TextIOWrapper name='seneca2.txt.checkscript' mode='r' encoding='UTF-8'>
  lab5bStudent.copy_file_add_line_numbers(filename, tempfile)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
/home/nyeh2/ops445/lab5/./CheckLab5.py:309: ResourceWarning: unclosed file <_io.TextIOWrapper name='seneca3.txt.checkscript' mode='w' encoding='UTF-8'>
  lab5bStudent.copy_file_add_line_numbers(filename, tempfile)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
ok
test_k_function_copy_file_add_line_numbers_correct_output_3 (__main__.lab5b)
[Lab 5] - [Investigation 1] - [Part 2] - Files - Test for output: ./lab5b.py ... /home/nyeh2/ops445/lab5/./CheckLab5.py:322: ResourceWarning: unclosed file <_io.TextIOWrapper name='seneca3.txt.checkscript' mode='r' encoding='UTF-8'>
  lab5bStudent.copy_file_add_line_numbers(filename, tempfile)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
/home/nyeh2/ops445/lab5/./CheckLab5.py:322: ResourceWarning: unclosed file <_io.TextIOWrapper name='seneca4.txt.checkscript' mode='w' encoding='UTF-8'>
  lab5bStudent.copy_file_add_line_numbers(filename, tempfile)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
ok
test_0 (__main__.lab5c)
[Lab 5] - [Investigation 2] - [Part 1] - Files - Test for file creation: ./lab5c.py ... ok
test_a (__main__.lab5c)
[Lab 5] - [Investigation 2] - [Part 1] - Files - Test for errors running: ./lab5c.py ... ok
test_a1 (__main__.lab5c)
[Lab 5] - [Investigation 2] - [Part 1] - Files - Test for correct shebang line: ./lab5c.py ... ok
test_a_function_add (__main__.lab5c)
[Lab 5] - [Investigation 2] - [Part 1] - Files - function add fails without 2 argument ... ok
test_b_function_read_file (__main__.lab5c)
[Lab 5] - [Investigation 2] - [Part 1] - Files - function read_file fails without 1 argument ... ok
test_c_function_add_correct_output_1 (__main__.lab5c)
[Lab 5] - [Investigation 2] - [Part 1] - Files - Test for output: ./lab5c.py ... ok
test_c_function_add_correct_output_2 (__main__.lab5c)
[Lab 5] - [Investigation 2] - [Part 1] - Files - Test for output: ./lab5c.py ... ok
test_c_function_add_correct_output_3 (__main__.lab5c)
[Lab 5] - [Investigation 2] - [Part 1] - Files - Test for output: ./lab5c.py ... ok
test_c_function_add_correct_output_4 (__main__.lab5c)
[Lab 5] - [Investigation 2] - [Part 1] - Files - Test for output: ./lab5c.py ... ok
test_d_function_read_file_correct_output_1 (__main__.lab5c)
[Lab 5] - [Investigation 2] - [Part 1] - Files - Test for output: ./lab5c.py ... /home/nyeh2/ops445/lab5/./CheckLab5.py:414: ResourceWarning: unclosed file <_io.TextIOWrapper name='seneca2.txt' mode='r' encoding='UTF-8'>
  self.assertEqual(lab5cStudent.read_file(filename), answer, msg=error_output)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
ok
test_d_function_read_file_correct_output_2 (__main__.lab5c)
[Lab 5] - [Investigation 2] - [Part 1] - Files - Test for output: ./lab5c.py ... ok

----------------------------------------------------------------------
Ran 41 tests in 0.038s

OK
